tensorflow_datasets/audio/freesound_audio_tagging_2019.py

```diff
@@ -0,0 +1,152 @@
+from pathlib import Path
```

Comment:
Tensorflow have own io library. Please check this [link](https://www.tensorflow.org/api_docs/python/tf/io).

Reply:
The Path library is nice but I am worried with Python2 compatibility.

If you want to use it, you would have to add it to the Python2 dependency list: https://github.com/tensorflow/datasets/blob/master/setup.py#L68